meetings/template.md

```diff
@@ -27,13 +28,14 @@
 - Technical Project updates
   - OpenVDB
   - OpenColorIO
+  - OpenEXR
+  - OpenCue
+  - OpenTimelineIO
```

Comment:
Nitpicky idea: it would be nice to randomize or rotate the project order every meeting. We tend to run out of time at the end of this list so keeping the same order means that the same projects get rushed every time.

Reply:
That's a great idea @bcipriano  -- definitely!